addons/kibana/6.8.x/kibana-8.yaml

```
@@ -78,20 +79,28 @@ spec:
               # Create index patterns
               for index in $INDICES; do
                 if [[ "$(curl -s -o /dev/null -w '%{http_code}\n' -L ${KB_URL}/api/saved_objects/index-pattern/${index})" != "200" ]]; then
-                  curl -XPOST "${KB_URL}/api/saved_objects/index-pattern/${index}" \
+                  RES_CREATE_INDEX=$(curl -XPOST -s -o /dev/null -w '%{http_code}\n' ${KB_URL}/api/saved_objects/index-pattern/${index} \
```

```suggestion
                  RES_CREATE_INDEX=$(curl -XPOST -s -o /dev/null -w '%{http_code}' ${KB_URL}/api/saved_objects/index-pattern/${index} \
```
the newline isn't necessary here (it doesn't hurt, anyway).

---

true true, but you missed two 😉